server/core/src/main/java/dev/slimevr/tracking/trackers/TrackerResetsHandler.kt

```diff
@@ -432,6 +453,28 @@ class TrackerResetsHandler(val tracker: Tracker) {
 		return totalMatrix.toQuaternion()
 	}
 
+	/**
+	 * Update the yaw reset smoothing time 
+	 */
+	@Synchronized
+	fun update() {
+		if (yawResetSmoothTimeRemain != 0.0f) {
+			var deltaTime = 0.001f
+			if (::fpsTimer.isInitialized) {
+				deltaTime = fpsTimer.timePerFrame
+			}
+			yawResetSmoothTimeRemain = max(0.0f, yawResetSmoothTimeRemain - deltaTime)
+			if (yawResetSmoothTimeRemain == 0.0f) {
+				yawFix = yawFixNew
+			} else {
+				yawFix = yawFixOld.interpR(
+					yawFixNew, 
+					animateEase( (yawResetSmoothTime - yawResetSmoothTimeRemain) / yawResetSmoothTime)
+				) 
```

Comment:
This might make it a bit less readable, but it would be simpler to just ease "backwards". Maybe a comment could also be added to explain this?
```suggestion
				yawFix = yawFixNew.interpR(
					yawFixOld,
					animateEase(yawResetSmoothTimeRemain / yawResetSmoothTime)
				) 
```

Reply:
Probably yes, I'll add some comment here when solving conflicts